app/_src/quickstart/kubernetes-demo.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Deploy Kuma on Kubernetes
+---
+
+To start learning how {{site.mesh_product_name}} works, you run and secure a simple demo application that consists of two services:
+
+- `demo-app`: a web application that lets you increment a numeric counter. It listens on port 5000
+- `redis`: data store for the counter
+
+## Prerequisites
+- [Helm](https://helm.sh/) - a package manager for Kubernetes
+- [Kind](https://kind.sigs.k8s.io/) - a tool for running local Kubernetes clusters
+
+## Start Kubernetes cluster
+
+Start a new Kubernetes cluster on your local machine by executing:
+
+```sh
+kind create cluster --name=kuma
+```
+
+{% tip %}
+You can skip this step if you already have a Kubernetes cluster running.
+It can be a cluster running locally or in a public cloud like AWS EKS, GCP GKE, etc.
+{% endtip %}
+
+## Install Kuma
+
+Install Kuma control plane with Helm by executing:
+
+```sh
+helm repo add kuma https://kumahq.github.io/charts
+helm repo update
+helm install --create-namespace --namespace {{site.mesh_namespace}} {{ site.kuma_install_name }} {{ site.mesh_helm_repo }}
+```
+
+## Deploy demo application
+
+1.  Deploy the application
+    ```sh
+    kubectl apply -f https://raw.githubusercontent.com/kumahq/kuma-counter-demo/master/demo.yaml
+    ```
+
+2.  Port forward the service to the namespace on port 5000:
+
+    ```sh
+    kubectl port-forward svc/demo-app -n kuma-demo 5000:5000
+    ```
+
+3.  In a browser, go to `127.0.0.1:5000` and increment the counter.
+
+The demo app includes the `kuma.io/sidecar-injection` label enabled on the `kuma-demo` namespace.
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kuma-demo
+  labels:
+    kuma.io/sidecar-injection: enabled
+```
+
+This means that {{site.mesh_product_name}} [already knows](/docs/{{ page.version }}/production/dp-config/dpp-on-kubernetes/) that it needs to automatically inject a sidecar proxy to every Kubernetes pod in the `kuma-demo` namespace.
+
+## Explore GUI
+
+You can view the sidecar proxies that are connected to the {{site.mesh_product_name}} control plane.
+
+{{site.mesh_product_name}} ships with a **read-only** GUI that you can use to retrieve {{site.mesh_product_name}} resources. By default, the GUI listens on the API port and defaults to `:5681/gui`.
```

Comment:
**Suggestion**: Not sure this needs to refer to the default GUI path. If it needs to, I’d mention that on line 77.

```suggestion
{{site.mesh_product_name}} ships with a **read-only** GUI that you can use to retrieve {{site.mesh_product_name}} resources. By default, the GUI listens on the API port which defaults to `5681`.
```